components/prism-vhdl.js

```diff
@@ -0,0 +1,24 @@
+Prism.languages.vhdl = {
+	'comment': /(--[^\r\n]+)/,
+	// support for all logic vectors
+	'vhdl-vectors': {
+		'pattern': /(\b[oOxXbB]"[\dA-Fa-f_]+"|"[UuXx01ZzWwLlHh-]+")/,
+		'alias': 'number'
+	},
+	'string': /(")(\\\n|\\?.)*?"/,
+	'constant': /\b(use|library)\b/i,
+	// support for predefined attributes included
+	'keyword': /\b(access|after|alias|all|architecture|array|assert|attribute|begin|block|body|buffer|bus|case|component|configuration|constant|disconnect|downto|else|elsif|end|entity|exit|file|for|function|generate|generic|group|guarded|if|impure|in|inertial|inout|is|label|library|linkage|literal|loop|map|new|next|null|of|on|open|others|out|package|port|postponed|procedure|process|pure|range|record|register|reject|report|return|select|severity|signal|shared|subtype|then|to|transport|type|unaffected|units|until|use|variable|wait|when|while|with|'base|'left|'right|'high|'low|'ascending|'image|'value|'pos|'val|'succ|'pred|'leftof|'rightof|'left|'left|'right|'right|'high|'high|'low|'low|'range|'range|'reverse_range|'reverse_range|'length|'length|'ascending|'ascending|'delayed|'stable|'stable|'quiet|'quiet|'transaction|'event|'active|'last_event|'last_active|'last_value|'driving|'driving_value|'simple_name|'instance_name|'path_name)\b/i,
```

Comment:
I think there are a lot of duplicates here (unless I'm missing something...?): `'left`, `'right`, etc.